README.md

```diff
@@ -25,199 +25,68 @@ The purpose of Amber is not to create yet another framework, but to take advanta
+#### macOS
 
-<!-- WIP: ##### For RedHat & CentOS
-- `sudo yum groupinstall 'Development Tools' `
-- `sudo yum install readline-devel sqlite-devel openssl-devel libyaml-devel gc-devel libevent-devel` -->
-
-Once you have these dependencies, you can build `amber` from source:
-
-
-```shellsession
-$ git clone git@github.com:amberframework/amber.git
-$ cd amber
-$ make
-$ make install
 ```
-
-##### For ArchLinux & derivates
-- Install the CLI from [AUR package](https://aur.archlinux.org/packages/amber/). Dependencies are automatically installed.
-- `yaourt -S amber`
-
-You should now be able to run `amber` in the command line.
-
-#### Mac OS X
-
-Best way to get `amber` on Mac OS X is via Homebrew:
-
-```shellsession
-$ brew install amberframework/amber/amber
+brew tap amberframework/amber
+brew install amber
 ```
 
+#### Linux
 
-Refer to [this link](https://brew.sh/) if you don't have homebrew installed.
-
-## Amber CLI Commands
-
-```shell
-$ amber --help
-
-Amber - Command Line Interface
-
-  The `amber new` command creates a new Amber application with a default
-  directory structure and configuration at the path you specify.
-
-  You can specify extra command-line arguments to be used every time
-  `amber new` runs in the .amber.yml configuration file in your project
-  root directory
-
-  Note that the arguments specified in the .amber.yml file does not affect the
-  defaults values shown above in this help message.
-
-  Usage:
-  amber new [app_name] -d [pg | mysql | sqlite] -t [slang | ecr] -m [granite, crecto] --deps
-
-Subcommands:
-  d         alias for deploy
-  database  # Performs database maintenance tasks
-  db        alias for database
-  deploy    # Provisions server and deploys project.
-  e         alias for encrypt
-  encrypt   # Encrypts environment YAML file. [env | -e --editor | --noedit]
-  exec      # Executes Crystal code within the application scope
-  g         alias for generate
-  generate  # Generate Amber classes
-  n         alias for new
-  new       # Generates a new Amber project
-  routes    # Prints all defined application routes
-  w         alias for watch
-  watch     # Starts amber development server and rebuilds on file changes
-  x         alias for exec
-
-Options:
-  -d, --database  # Preconfigure for selected database. Options: pg | mysql | sqlite
-                  (default: pg)
-  -m, --model     # Preconfigure for selected model. Options: granite | crecto
-                  (default: granite)
-  -t, --template  # Preconfigure for selected template engine. Options: slang | ecr
-                  (default: slang)
-  -h, --help      # Describe available commands and usages
-  -v, --version   # Prints Amber version
-
-Example:
-  amber new ~/Code/Projects/weblog
-  This generates a skeletal Amber installation in ~/Code/Projects/weblog.
+```
+git clone https://github.com/amberframework/amber.git
```

Comment:
This is wrong as it will clone master instead of stable which isn't what what we should be recommending. I've changed this in the past. It should be.

```
curl -OL https://github.com/amberframework/amber/archive/stable.tar.gz
tar -xzf stable.tar.gz
cd amber-stable
make
```

Reply:
@elorest Ok, I will change it, but why not:

```
git clone https://github.com/amberframework/amber.git
cd amber
git checkout stable
sudo make install
```

It looks cleaner for me, WDYT?

Reply:
Either way. The git way is a bit more overhead and forces people to have git. Most of them will though.

Reply:
I prefer the git way :sweat_smile: [Is similar how lucky does it](https://luckyframework.org/guides/installing/#install-lucky)